ibtk/src/math/fortran/vclaplace2d.f.m4

```diff
@@ -159,3 +159,92 @@ c
       return
       end
 c
+ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+c
+c     Computes (f0,f1) = alpha grad lambda (div (u0,u1)) + beta (v0,v1).
+c
+c     Computes the side-centered variable coefficient dilatational stress force, 
+c     with cell-centered bulk viscosity coefficient lambda and side-centered
+c     vector fields (u0,u1) and (v0,v1).
+c
+ccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+c
+      subroutine stosvcdilatational2d(
```

Comment:
Let's call this the "grad div" operator unless there is a widely used name for this thing.